### ISSUE TITLE
Persistent queues

### ISSUE THREAD

**Issue by thavidu** [Azure/Azurite#77]
Can we please make queues persistent, just like blobs and tables?
If theres ever power loss or we stop docker, everything is currently gone.

**Comment by swissarmykirpan** [comment]
@XiaoningLiu this is a duplicate of #65

**Comment by XiaoningLiu** [comment]
Yes, we are internally discussing about a new architecture of Azurite, in the new architecture, the persistence layer could be injected. But it still needs time. : )
